### ISSUE TITLE
[Feature] Select parent paths

### ISSUE THREAD

**Issue by m-sundman** [qarmin/czkawka#663]
It would be useful to select paths that are parents of another path in the same group. I.e., if you have a group with files with these paths:
- /pics/2013
- /pics/2013/12
- /pics/2013/12/31
- /music/mp3s
- /music

Then it would select the first 2 and the last.

**Comment by m-sundman** [comment]
The rationale for this is that people often organize things into subfolders, and the further down into subfolders something is the more specifically organized that file is, so it's often that file that one wants to keep.

**Comment by chefkoch** [comment]
Yep, a "Select"-menu item by path, without typing it manually would be nice, which could be the root (defined search paths) in general and in addition their subfolders as well.

In AllDupe i.e. you can select all file in path x or y 
![2023-08-21 15_38_59-AllDup 4 5 26 PE - Search Result 'File content'](https://github.com/qarmin/czkawka/assets/601866/eac06d3e-2454-41f9-b7da-5290b3aec49b)
or click the ... which opens a dialog presenting you the subfolders of this path.
![2023-08-21 15_39_15-Select a folder_](https://github.com/qarmin/czkawka/assets/601866/c273d97a-1c45-4c8b-aada-53b6d46efa10)


This way you can "browse" the path which you would like to select in all groups, instead of the need to type the path manually in text box:
![2023-08-21 15_40_02-Czkawka – Mozilla Firefox](https://github.com/qarmin/czkawka/assets/601866/d3edad2e-f1ad-43ec-a743-bb5a992e1ce6)

**Comment by qarmin** [comment]
Custom paths are implemented in Krokiet

**Comment by m-sundman** [comment]
[qarmin](https://github.com/qarmin)
> Custom paths are implemented in Krokiet

Maybe I'm just dumb, but I'm unable to get Krokiet to select parent paths in a group.
I can select the longest path and invert, but that's not at all the same thing. The longest path might not be a subpath of any other in the group.

I don't quite understand what [chefkoch](https://github.com/chefkoch) is talking about, but it doesn't seem to have anything to do with my feature request. At least as far as I can tell.

**Comment by m-sundman** [comment]
@qarmin Since custom paths have nothing to do with this issue, AFAICT, could you reopen this?
